packages/colors/package.json

```diff
@@ -3,9 +3,16 @@
     "version": "3.0.0",
     "description": "Blueprint color definitions",
     "main": "lib/index.js",
+    "style": "lib/css/colors.css",
```

Comment:
this doesn't produce any CSS, so we don't need it

Reply:
ah i see, i noticed that there was a `lib/css/colors.css` that was created but is empty except for the source mapping. wasn't sure if that required adding this or not